os/linux/gbmbufferhandler.h

```diff
@@ -49,6 +49,9 @@ class GbmBufferHandler : public NativeBufferHandler {
   uint32_t GetFd() const override {
     return fd_;
   }
+  bool GetInterlace(HWCNativeHandle handle) const override {
+    return false;
+  }
```

Comment:
Is "handle" included for future use? It is not being referenced here.

Reply:
Same with above